README.md

```diff
@@ -106,3 +186,185 @@ git secrets --scan /path/to/file (/other/path/to/file *)
 ```Shell
 git secrets --scan -r /path/to/directory
 ```
+
+## API Server Configuration
+
+Spring application configs, in `application.properties` files, specify behavior
+like logging. They are static files bundled with the built Java binary.
+
+Database connection information is read from `application-web.xml`. These
+secrets vary per-environment; Ruby setup scripts pull the values from Google
+Cloud Storage and generate the XML, which is then deployed with the Java binary.
+
+Server behavior configuration is stored in the database. It may be changed
+without restarting the server, but is usually set only at deployment time. It's
+based on config\_$ENV.json files (which are converted into `WorkbenchConfig`
+objects) and loaded into the database by `workbench.tools.ConfigLoader`.
+
+`CacheSpringConfiguration`, a Spring `@Configuration`, provides
+the `@RequestScoped` `WorkbenchConfig`. It caches the values fetched from the
+database with a 10 minute expiration.
+
+## API Server Database Updates
```

Comment:
I think this stuff will be out of date - can you put a TODO here or else just remove for now